src/plugins/kibana/public/discover/directives/timechart.js

```diff
@@ -17,7 +17,7 @@ uiModules
       const init = function () {
         // This elem should already have a height/width
         const myChart = new vislib.Chart(elem[0], {
-
+          addLegend: false
```

Comment:
Yes. We already have an addLegend parameter that isn't used due to @rashidkpc changes to the legend. On the discover page, we don't need the legend and therefore adding the option for addLegend back is an easy way to remove it.